### PR TITLE
Data services update

### PR DESCRIPTION
**Pull request by mbertrand**
Fixes broken provisioning caused by outdated diff patch
Sets GeoNode branch to 2.4.x (more stable)
Updates GS_TMP_DIR setting to avoid permissions problems

### CODE REVIEW

**Comment by mgrauer** [comment]
Worked for me @mbertrand , thanks.  I'll need to fix this failing test (unrelated to these changes) before it can go in.

File: data_services/deploy/ansible/roles/geonode/tasks/main.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@
   copy: src=known_hosts dest={{ssh_dir}}/known_hosts
 
 - name: checkout latest web app code
-  git: repo={{code_repository}} version={{app_version}} dest={{app_code_dir}}/{{app_name}} force=yes
+  git: repo={{code_repository}} version=2.4.x dest={{app_code_dir}}/{{app_name}} force=yes
```
Comment (aashish24) [inline data_services/deploy/ansible/roles/geonode/tasks/main.yml]: Why hardcode to 2.4.x?

**Comment by aashish24** [comment]
@mbertrand @mgrauer if the failing tests are not related to this branch then why wait for merging? Just curious.

**Comment by aashish24** [comment]
Thanks @mbertrand for pushing a fix so quickly.

File: data_services/deploy/ansible/roles/geonode/tasks/main.yml (hunk 1 above)
Comment (mbertrand) [inline data_services/deploy/ansible/roles/geonode/tasks/main.yml]: 2.4.x is more stable (2.4 is the latest official release).  Master could
introduce database schema changes or other updates that are incompatible
with dataqs and/or existing Geonode instances.

Comment (aashish24) [inline data_services/deploy/ansible/roles/geonode/tasks/main.yml]: Sounds good so is the app_version is useless>?

Comment (mbertrand) [inline data_services/deploy/ansible/roles/geonode/tasks/main.yml]: Sorry, I overlooked that; app_version is the better place to set branch to 2.4.x.  I updated the code and also added several more data's processors to run (gdacs and usgs_quakes).

**Comment by mgrauer** [comment]
:snowflake: :snowflake:

**Pull request merged**